src/content/energy-consumption/index.md

```diff
@@ -7,61 +7,42 @@ sidebar: true
-
-Similarly to proof-of-work, a malicious entity would require at least 51% of the total ETH staked in the network to execute a [51% attack](/glossary/#51-attack). However, unlike on proof-of-work, where the potential loss of a failed attack is only the cost of generating the hash power needed to mine, on proof-of-stake, the possible loss of an attack is the entire amount of ETH used as collateral. This disincentive structure allows for network security with proof-of-stake while eliminating the need to expend energy on arbitrary computations. Detailed explanations of the network security under proof-of-stake can be found [here](/developers/docs/consensus-mechanisms/pos/) and [here](https://vitalik.ca/general/2017/12/31/pos_faq.html).
+Since inception, Ethereum has aimed to implement a PoS consensus mechanism, but doing this without sacrificing security and decentralization has taken years of focused research and development. Therefore, the network started by using PoW consensus. PoW consensus requires miners to use their computing hardware to solve a puzzle. The solution to this puzzle acts as proof that some energy has been expended by the miner which in turn demonstrates that they have invested some real-world value for the right to add to the blockchain. PoW is just a mechanism to decide who gets to add the next block, as is PoS. Swapping PoW for PoS, where the real world value invested comes from ether staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.
 
 ## The Merge {#the-merge}
 
-There is a functional proof-of-stake chain called the [Beacon Chain](/upgrades/beacon-chain/) that has been running since December 2020 that is demonstrating the viability of the proof-of-stake protocol. The Merge refers to the point in time when Ethereum leaves proof-of-work behind and fully adopts proof-of-stake. The Merge is expected to happen ~Q3/Q4 2022. [More on The Merge](/upgrades/merge/).
+A PoS blockchain known as the "Beacon Chain" has been running since November 2020 alongside the PoW Ethereum Mainnet. In 2022 several custom devnets have been merged with their own Beacon Chains to aid in debugging Ethereum clients in advance of the real merge. More recently, Ethereum developers have started merging public testnets (Kiln, Kintsugi, Ropsten) and shadow-forks (forks of Ethereum Mainnet) as the final stage of testing. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's PoW mining will be switched off, PoS consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.
```

Comment:
Is it worth mentioning Goerli and Sepolia in some capacity here?

Reply:
I would maybe emphasize the PoS chain only ran the consensus algorithm, and didn't run user transactions. "PoS blockchain" sort of implies another version of Ethereum was launched. 

> In 2022 several custom devnets have been merged with their own Beacon Chains to aid in debugging Ethereum clients in advance of the real merge.

I would rephrase this to: 

> In 2022, several proof-of-work devnets were launched and transitioned to proof-of-stake by merging with their own Beacon Chains. These helped client teams test software before moving to longer-lived networks. 


> More recently, Ethereum developers have started merging public testnets (Kiln, Kintsugi, Ropsten) and shadow-forks (forks of Ethereum Mainnet) as the final stage of testing.

Similarly, I'd suggest:

> After these devnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple [shadow forks](https://etherworld.co/2022/04/20/ethereum-mainnet-shadow-forking-an-overview/) of existing testnets and mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake.

Reply:
thanks, updated accordingly